tests/test1165.pl

```diff
@@ -67,9 +67,9 @@ sub scanconf_cmake {
     my ($f)=@_;
     open S, "<$f";
     while(<S>) {
-        if(/(CURL_DISABLE_[A-Z_]+)/g) {
+        if(/(?:^|[^A-Za-z0-9\-])(CURL_DISABLE_[A-Z0-9_]+)(?:$|[^A-Za-z0-9\-])/g) {
```

Comment:
Same here

Reply:
It works as well.
I don't create PRs without proper local testing.
I added debug printing of the results and results are fine.